Configurations/10-main.conf

```diff
@@ -1366,6 +1378,7 @@ my %targets = (
                                       return [ @defs ];
                                   }),
         lib_cflags       => add("/Zi /Fdossl_static"),
+        lib_defines      => add([ "L_ENDIAN" ]),
```

Comment:
On *side* note, one can actually move L_ENDIAN to template, *all* Windows are little-endian.

Reply:
You mean to `BASE_windows`?  Yeah, sure, although I think that belongs in a separate cleanup PR.